timestamp/http_test.go

```diff
@@ -75,7 +75,7 @@ func TestHTTPTimestampGranted(t *testing.T) {
 		t.Fatalf("NewHTTPTimestamper() error = %v", err)
 	}
 	message := []byte("notation")
-	req, err := NewRequestFromBytes(message)
+	req, err := NewRequestWithData(message, crypto.SHA256)
 	if err != nil {
 		t.Fatalf("NewRequestFromString() error = %v", err)
```

Comment:
Method name on line 80 does not match line 78. Same for multiple occurrences in this file.

Reply:
NewRequestFromString is removed but multiple error messages reference it.